app/models/google_event.rb

```diff
@@ -111,6 +105,20 @@ def load_emails
       ConferenceRoom.pluck(:email)
     end
 
+    GOOGLE_EVENT_DECLINED_RESPONSE = 'declined'.freeze
+    def add_events_from_room(room, service, events, config)
+      service.list_events(room.email, config) do |result, _|
+        if result
```

Comment:
When we do not have permission from Google to load events from a given calendar.